dust_extinction/meta.yaml

```diff
@@ -0,0 +1,38 @@
+{% set name = 'dust_extinction' %}
+{% set version = '0.7' %}
+{% set number = '0' %}
+
+about:
+    home: https://github.com/karllark/{{ name }}
+    license: BSD
+    summary: |
+        Provides models of interstellar dust extinction curves
+
+build:
+    number: {{ number }}
+
+package:
+    name: {{ name }}
+    version: {{ version }}
+
+requirements:
+    build:
+    - astropy >=1.3
```

Comment:
Astropy 1.3 is pretty old, are you sure you don't need to bump this up?

Reply:
What would be a reasonable version?  I don't want to shut people out, but also am not sure what version this package will work with.  My travis CI runs test the astropy lts and latest versions.

Reply:
LTS is acceptable minversion if you are testing against it. I just think that 1.3 might be too ambitious.

Reply:
Can you be more explicit?  What should I put in instead of ">=1.3"?

Reply:
`>=2` ?

Reply:
Can do.  Why the ?

Reply:
Because you make the final call. I am just merely suggesting. 😉  If you insist that it works with `astropy>=1.3`, then be prepared for bug reports when people try to use it with `astropy=1.3` and find problems.